examples/MDFTestingExample/MDFTestingExample.xcodeproj/project.pbxproj

```diff
@@ -187,7 +187,7 @@
 		DD871046233A7EFD0014F4B7 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 1100;
+				LastUpgradeCheck = 1120;
```

Comment:
Are these changes required?  Do we need to lower the deployment target from 13 to 10 to upgrade to Bazel 2.0.0?